ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_order_statistics.py

```diff
@@ -81,3 +81,38 @@ def test_numpy_nanpercentile(
         test_flags=test_flags,
         input_dtypes=input_dtypes,
     )
+
+
+def test_numpy_nanquantile(

```

Comment:
Could you please wrap this with proper wrapper functions like in `test_numpy_nanpercentile`?

Reply:
wrapped!